### DISCUSSION TITLE
No "Most similar known cluster" output

### DISCUSSION THREAD

**Discussion by caizhangbin** [antismash/antismash#665] [Q&A]
I successfully installed and executed the software, but I encountered an issue where it didn't generate 'Most similar known cluster' results. when I used the web server to analyze the same samples, it produced the expected results. The 'Region' and 'Type' results remained consistent

**Comment by SJShaw** [comment]
The website default options are not the command line default options.

To get the known cluster results, you need to specifically enable the known clusterblast analysis with `--cb-known`.

To see all the available analyses and simple options, use `--help`. For all possible options, use `--help-showall`.
